webpack.config.js

```diff
@@ -105,14 +106,18 @@ const commonConfig = (env) => {
       maxEntrypointSize: 30000000,
     },
     resolve: {
-      // Required for github.dev and `minimatch`, as Webpack 5 doesn't add polyfills automatically anymore.
+      // Required for github.dev, `minimatch` and vscode-redhat-telemetry, as Webpack 5 doesn't add polyfills automatically anymore.
       fallback: {
         path: require.resolve("path-browserify"),
         os: require.resolve("os-browserify/browser"),
         fs: false,
         child_process: false,
         net: false,
         buffer: require.resolve("buffer/"),
+		assert: false,
+		process: false,
+		stream: false,
+		util: false
```

Comment:
based on https://coreos.slack.com/archives/CK14U2L4V/p1666101124417369?thread_ts=1666099245.975149&cid=CK14U2L4V maybe need to change that